packages/firebase_database/firebase_database_web/lib/firebase_database_web.dart

```diff
@@ -67,8 +67,8 @@ class FirebaseDatabaseWeb extends DatabasePlatform {
   }
 
   @override
-  Future<void> setLoggingEnabled(bool enabled) {
-    throw UnimplementedError("setLoggingEnabled() is not supported for web");
+  Future<void> setLoggingEnabled(bool enabled) async {
+    firebase.enableLogging(enabled);
```

Comment:
@lesnitsky Please check this. JS has a top-level logging function. So I used it.
https://firebase.google.com/docs/reference/js/v8/firebase.database#enablelogging

Reply:
We want to use `firebase.database.enableLogging(enabled);` here instead; see https://firebase.google.com/docs/reference/node/firebase.database#enablelogging

Reply:
Then we need to create that function in our interop

Reply:
I tried by adding the `enableLogging` function in the `Database` class. But integration test throws the error:

Reply:
Let's stick with a `UnimplementedError` for now - I don't think the one at the root is right but let's not block this PR - can you leave a TODO in the code about it also - thanks

Reply:
done

Reply:
@Salakar enableLogging from `firebase.database` is namespaced under `firebase.database` (which is handled by @JS('firebase.database') annotation). It's not a method of a `Database` class. Implemented in d8ffec0